content/docs/learning-guide/tutorials/custom-tools/shape-example-py.md

```diff
@@ -0,0 +1,470 @@
+
+5. Open O3DE Editor for your project.
+
+6. Open the **Python Scripts** panel by selecting **Tools > Other > Python Scripts** from the file menu. (See A in the following image.)
+
+7. Open `pyshapeexample_dialog` in the Python Scripts panel. (See B in the following image.)
+
+8. Alternatively, open the tool directly by clicking on the tool's icon in the **Edit Mode Toolbar**. (See C in the following image.)
+
+Now you can access the Shape Example tool! By default, this tool contains a simple user interface (UI). In the next steps, we'll design the tool's UI and code its functionality. (See D in the following image.) 
+
+![O3DE Editor with a tool created using the PythonToolGem template](/images/learning-guide/tutorials/custom-tool/shape-example/python-shape-example-in-editor.png)
+
+## Code directory
+
+In the ShapeExample Gem's `Code/Source` and `Editor/Scripts` directories, you'll find the following classes and frameworks that make up your custom tool.
+
+### Modules and system components
+
+All Gems have modules and system component classes written in C++ that connect the tool to O3DE and allow it to communicate with other systems. The `PythonToolGem` template already contains all of the code needed to run the tool in the O3DE Editor. You can find these C++ files in `Code/Source`.
+
+For more information on Gem modules and system components, refer to the [C++ Programming for Gem Development](docs/user-guide/gems/development/programming-for-gems.md) page.
+
+### O3DE and Qt frameworks
+
+O3DE extends the **Qt** framework, so you will use [PySide2](https://pypi.org/project/PySide2/)---a Python module based on [Qt for Python](https://doc.qt.io/qtforpython/)---to create a graphical user interface (GUI). You will also use O3DE's EBuses to communicate with other O3DE interfaces, such as entities and components, and connect them to Qt elements.
+
+You will write most of your tool's functionality and UI elements in the`PyShapeExampleDialog` class that's located in `Editor/Scripts/pyshapeexample_dialog.py`.
+
+### Qt Resources
+
+The [Qt Resource System](https://doc.qt.io/qt-5/resources.html) allows Gems to store and load image files via a `.qrc` file. This eliminates the need to load image files from absolute paths, making it simpler for you to distribute your Gem. Later, you will store an image file to create an icon for your tool.
+
+### Dialogs, Widgets, and layouts
+
+With Qt for Python, you can create a dialogs which are top-level windows. 
+
+Qt for Python creates a top-level window called a *dialog*. Within a dialog are *widgets*, which are containers for UI elements, and *layouts*, which define how those UI elements are arranged. (In comparison, Qt in C++ creates a main widget instead of a dialog.) Each widget can have its own layout and additional sub-widgets. This nested widget and layout structure allows you to organize groups of UI elements.
```

Comment:
We might want to discuss changing this to just be a QWidget as well instead of a QDialog for consistency. @HogJonny-AMZN